src/js/worker/api/actions.js

```diff
@@ -138,6 +138,17 @@ const playMenu = {
         }
     },
 
+    throughRound: async (conditions: Conditions) => {
+	if (g.phase === PHASE.PLAYOFFS) {
+	    const playoffSeries = await idb.cache.playoffSeries.get(g.season);
+
+            const series = playoffSeries.series[playoffSeries.currentRound][0];
+            const numDaysThisSeries = 7 - series.home.won - series.away.won;
+
+            game.play(numDaysThisSeries, conditions);
```

Comment:
@dumbmatter - this logic isn't quite right.  

The way that `numDaysThisSeries` is calculated is by using the first playoff series.  It works fine if applied at the beginning of the round, but if if is not the Finals (i.e. multiple series are occurring) and a few games in the round have been played via `(Play) One day` and the series results are not even (e.g. it is the conference finals and one series is 1-1 and the other is 2-0) it sometimes plays an extra game into the next round.

I don't think I can take the max value of `numDaysThisSeries` for all rounds - that will lead to the same issue.  I'm guessing that I would need to play one game at a time and watch for the `currentRound` to change.  I tried that approach and was getting some errors so I decided to submit this first to see if this is a UI change you like, and then I can spend more time on it.